CMSgov/mint-app
# [EASI-3031] Filter Banner and Summary Updates

# EASI-3031

## Changes and Description

- Replacing the filter box with the filter banner that currently displays on the filtered content. 
- Update "Back to all models" link to "View all models"
- Updates to "Show model summary" and "Hide model summary" link icon and placement
- The overall status should now be incorporated into the summary below the name of the model
- A new "Edit Model Plan" link should show near the last updated information. 
    - Because of this change, we'll also need to update the side nav of the task list. Add a new "Actions" heading and a "Switch to the Read View for this Model Plan" link that takes the user to the read view. 

![image](https://github.com/CMSgov/mint-app/assets/2349702/d3afa1ca-653b-4c57-9951-646d20421028)





## How to test this change

1. Navigate to a task list
2. Check side navigation has "Actions" and new link to "Switch to Read View for this Model Plan"
3. Ensure new header looks like the figma


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
